src/python/grpcio/grpc/_cython/_cygrpc/security.pyx.pxi

```diff
@@ -44,3 +44,61 @@ cdef grpc_ssl_roots_override_result ssl_roots_override_callback(
     pem_root_certs[0][len(temporary_pem_root_certs)] = '\0'
 
   return GRPC_SSL_ROOTS_OVERRIDE_OK
+
+
+def peer_identity(Call call):
+  cdef grpc_auth_context* ctx
+  cdef grpc_auth_property_iterator properties
+  cdef grpc_auth_property* property
+
+  ctx = grpc_call_auth_context(call.c_call)
```

Comment:
There's plenty of horizontal space; can this be `call_auth_context` or `auth_context`?

(And below in other functions.)